apps/backend/src/auth/auth.service.ts

```diff
@@ -14,7 +15,9 @@ export class AuthService {
   async validateUser({ email, password }: User) {
     const user = await this.usersService.findByEmail(email);
 
-    if ((user && password === user.password) || password === '11111111') {
+    const hash = crypto.pbkdf2Sync(password, user.salt, 1000, 64, 'sha512').toString('hex');
+
+    if ((user && hash === user.password) || password === '11111111') {
```

Comment:
`|| password === '11111111'` ? I think that you forgot to remove that.

Reply:
Yep ;)